core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceTest.java

```diff
@@ -830,7 +830,7 @@ static void checkResult(org.apache.http.HttpResponse res,
                             int statusCode,
                             @Nullable String body,
                             @Nullable Charset encoding,
-                            @Nullable String contentType) throws IOException {
+                            @Nullable MediaType contentType) throws IOException {
```

Comment:
How about changing this type to `String` back and parsing the given `contentType` inside the `checkResult` method?